source/solvers/navier_stokes_assemblers.cc

```diff
@@ -1594,12 +1593,10 @@ LaplaceAssembly<dim>::assemble_matrix(
 
               // Laplacian on the velocity terms
```

Comment:
I know Lucka explain to me why viscosity term was removed, but it would be nice to explain why the units are not consistent now